scripts/validate_releases.sh

```diff
@@ -36,10 +36,13 @@ EXPECTED_XCODEBUILD_RESULTS=( \
     [2.5.2]=${ERROR_CODE} \
     [2.5.3]=${SUCCESS_CODE} \
     [2.6.0]=${SUCCESS_CODE} \
-    [2.6.1]=${SUCCESS_CODE} \
-    [2.7.0]=${SUCCESS_CODE} \
-    [2.7.1]=${SUCCESS_CODE} \
-    [2.7.2]=${SUCCESS_CODE}
+    [2.6.1]=${ERROR_CODE} \
+    [2.7.0]=${ERROR_CODE} \
+    [2.7.1]=${ERROR_CODE} \
+    [2.7.2]=${ERROR_CODE} \
```

Comment:
Resolving dependencies is expected to fail for `v2.7.2` from now on. `v2.7.3` patch will fix this issue.

Reply:
Should we add there also`SUCCESS_CODE` for `v2.5.4`, `2.6.2`, `2.7.3`?  Will them be added as part of a fix release?

Reply:
Still awaiting for an approval to release these versions. When they're out, I'll update this list.